float64 therm1
float64 therm2
float64 therm3
